Week7/solution 7

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+
+#1: Add 12 Records. Take input from the user.
+data = []
+for _ in range(12):
+    prod_no = input("Enter Product Number: ")
+    prod_name = input("Enter Product Name: ")
+    sales = [int(input(f"Enter sales for {month}: ")) for month in ["Jan", "Feb", "Mar", "Apr", "May", "Jun"]]
+    data.append([prod_no, prod_name] + sales)
+
+#2: Create a DataFrame.
+df = pd.DataFrame(data, columns=["Prod_No", "Prod_Name", "Jan", "Feb", "Mar", "Apr", "May", "Jun"])
+
+#3: Change Column Names.
+df.columns = ["Product No", "Product Name", "January", "February", "March", "April", "May", "June"]
+
+#4: Add columns for "Total Sell" and "Average Sell".
+df["Total Sell"] = df.iloc[:, 2:].sum(axis=1)
+df["Average Sell"] = df.iloc[:, 2:8].mean(axis=1)
```

Comment:
why did you use axix? and what is used for?

Reply:
axis=0,it means that the operation will be applied vertically along the rows.
axis=1,it means that the operation will be applied horizontally along the columns.